manifests/init.pp

```diff
@@ -20,15 +20,18 @@
 #         runas: root
 #         passwd: true
 #
+# @param include_dirs an array of paths to include in the sudoers file
+#
 # @param package_ensure The ensure status of packages to be managed
 #
 # @author https://github.com/simp/pupmod-simp-sudo/graphs/contributors
 #
 class sudo (
-  Hash   $user_specifications = {},
-  Hash   $default_entries     = {},
-  Hash   $aliases             = {},
-  String $package_ensure      = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),
+  Hash                        $user_specifications = {},
+  Hash                        $default_entries     = {},
+  Hash                        $aliases             = {},
+  String                      $package_ensure      = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),
+  Array[Stdlib::Absolutepath] $include_dirs        = [],
```

Comment:
You may want to allow this to accept a `Hash` if you also want to be able to prevent bad/unauthorized updates on the system.